pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -2430,3 +2434,30 @@ func (a *ApplicationBackupController) createVMIncludeResources(backup *stork_api
 func IsBackupObjectTypeVirtualMachine(backup *stork_api.ApplicationBackup) bool {
 	return backup.Spec.BackupObjectType == resourcecollector.PxBackupObjectType_virtualMachine
 }
+
+func (a *ApplicationBackupController) pruneNamespacesForVmBackup(backup *stork_api.ApplicationBackup) bool {
+	nsList := make([]string, 0)
+	if len(backup.Spec.Namespaces) == 0 {
+		return false
+	}
+	if len(a.vmIncludeResource[string(backup.UID)]) != 0 {
```

Comment:
nit: Empty line after `if` statement, can we remove it to keep uniform across the code.